packages/qwik/src/core/components/prefetch.ts

```diff
@@ -30,6 +30,14 @@ export const PrefetchServiceWorker = (opts: {
   fetchBundleGraph?: boolean;
   nonce?: string;
 }): JSXNode<'script'> => {
+  const isTest = import.meta.env.TEST;
```

Comment:
What's the point of this?

Reply:
We have few test for this component and we need to simulate the production mode even if we are testing